ert_gui/gert_main.py

```diff
@@ -62,20 +64,18 @@ def run_gui(args):
     # be the base name of the original config
     args.config = os.path.basename(args.config)
     ert = EnKFMain(res_config, strict=True, verbose=args.verbose)
-    with Storage.connect_or_start_server(
-        res_config=os.path.basename(args.config)
-    ) as storage:
+    with Storage.connect_or_start_server(res_config=os.path.basename(args.config)):
         # window reference must be kept until app.exec returns
         notifier = ErtNotifier(args.config)
-        window = _start_window(ert, notifier, args)
+        _start_window(ert, notifier, args)
```

Comment:
not sure if this is OK given the comment `# window reference must be kept until app.exec returns` I am sure it is fine but if it is not then we will get some very strange errors that are hard to debug, so I would keep 
```
window = _start_window(ert, notifier, args) # noqa
```

Reply:
This is a mixup with the flake8 commit, where this change has been reverted. Should go away when rebased on main.

Reply:
https://github.com/equinor/ert/commit/b06303d5f2011495da1e70f7da972cc4ad0e1998

Reply:
Not needed at all, remove if you wish.

Reply:
ok, I thought it was some obscure new flake8 thing :smile: